Widgets/RoomView.cpp

```diff
@@ -66,14 +70,18 @@ void RoomView::paintBackgrounds(QPainter& painter, Room* room, bool foregrounds)
   for (auto bkg : room->backgrounds()) {  //TODO: need to draw last if foreground
     if (!bkg.visible() || bkg.foreground() != foregrounds) continue;
     ProtoModel* bkgRes = MainWindow::resourceMap->GetResourceByName(TreeNode::kBackground, bkg.background_name());
+    if (bkgRes == nullptr) {
+      continue;
+    }

```

Comment:
This one is fine, but put it on a single line. Do NOT defensively log this one, since it's not an editor problem, it's _valid_ for the project to be in this state.
`if (bkgRes == nullptr) continue;`